cogs/welcome/welcome.py

```diff
@@ -537,6 +648,112 @@ def check(message: discord.Message):
         await self.config.guild(ctx.guild).get_attr(key).set(greetings)
         return
 
+    # [p]welcomeset greetings image
+    @greetings.group(name="image")
+    async def image(self, ctx: Context):
+        """Base command for the image command group"""
+        pass
```

Comment:
No need `pass` if there's already a docstring. We only need `pass` when the body consists of nothing other than comments.